anchore_engine/services/apiext/swagger/swagger.yaml

```diff
@@ -376,78 +376,78 @@ paths:
   /images:
     post:
       tags:
-      - Images
+        - Images
       x-swagger-router-controller: anchore_engine.services.apiext.api.controllers.images
       operationId: add_image
       x-anchore-authz-action: createImage
       summary: Submit a new image for analysis by the engine
       description: Creates a new analysis task that is executed asynchronously
       parameters:
-      - name: force
-        in: query
-        type: boolean
-        required: false
-        description: Override any existing entry in the system
-      - name: autosubscribe
-        in: query
-        type: boolean
-        required: false
-        description: Instruct engine to automatically begin watching the added tag for updates from registry
-      - name: image
-        required: true
-        in: body
-        schema:
-          $ref: "#/definitions/ImageAnalysisRequest"
-      - $ref: "#/parameters/AsAccountParameter"
+        - name: force
+          in: query
+          type: boolean
+          required: false
+          description: Override any existing entry in the system
+        - name: autosubscribe
+          in: query
+          type: boolean
+          required: false
+          description: Instruct engine to automatically begin watching the added tag for updates from registry
+        - name: image
+          required: true
+          in: body
+          schema:
+            $ref: "#/definitions/ImageAnalysisRequest"
+        - $ref: "#/parameters/AsAccountParameter"
       responses:
         200:
           description: Successfully added image to analysis queue
           schema:
-            $ref: "#/definitions/AnchoreImageList"
+            $ref: "#/definitions/AnchoreImage"
```

Comment:
Placeholder for checking if this impacts clients

Reply:
Controller logic backing this API returns a list object. So the api definition *was* aligned with the response prior to this change. Whether the api should return a single object or a list is a different question, one that needs to be addressed across the board

https://github.com/anchore/anchore-engine/blob/0e393e6f13e18cb0e0c0f81b199295da732eb054/anchore_engine/services/apiext/api/controllers/images.py#L1412

Reply:
the plot thickins...

Reply:
reverted back to the original